src/main/webapp/filterScript.js

```diff
@@ -0,0 +1,39 @@
+/**
+* Turn a filter a different color when pressed, and change class accordingly.
+**/
+function turnBlueWhenClicked() { //eslint-disable-line
+  const allFilters = document.querySelectorAll('.unclicked-filter');
+  for (let i = 0; i < allFilters.length; i++) {
+    allFilters[i].onclick = function() {
+      const currFilter = allFilters[i];
+      if (currFilter.classList.contains('unclicked-filter')) {
```

Comment:
Since 'unclicked-filter' and 'clicked-filter' are used multiple times in this file, I suggest replacing these with constants.